1028-spinner-design-guidance.md

```diff
@@ -0,0 +1,22 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-11
+updated: 2024-10-22
+---
+
+# Spinner Design Guidance
+
+A [spinner](https://clarity.design/documentation/spinner) component is a visual indicator of an ongoing, user-initiated process.
+
+## Guidance
+
+- **Must** use page spinners for tracking the progress of an operation related to an entire page.
+- **Must** use inline spinners for tracking the progress of an operation related to a specific component.
+- **Must** place the spinner where you want to focus the user’s attention when the process completes.
+- **Should** provide a brief description of the process, for example, “loading …”.
+- **May** use progress bar as an alternative to spinner, if more space is availble. 
```

Comment:
```suggestion
- **May** use progress bar as an alternative to spinner, if more space is available.
```

I think this should something about using progress bars is the progress status is known instead of indeterminate. cc:@jbusta5

Reply:
what about: 

Design teams **May** use a progress bar as an alternative to a spinner if more space is available. Progress bars are preferable when the progress status is known, while spinners are better suited for indeterminate states.

@kevinbuhmann